test/lvol/fragmap.sh

```diff
@@ -9,44 +9,197 @@ source $rootdir/test/common/autotest_common.sh
 source $rootdir/test/lvol/common.sh
 source $rootdir/test/bdev/nbd_common.sh
 
-function test_fragmap() {
+NUM_CLUSTERS=10
+LVS_DEFAULT_CLUSTER_SIZE_BTYE=$((LVS_DEFAULT_CLUSTER_SIZE_MB*1024*1024))
+
+function verify() {
+	local fragmap="$1"
+	local expected_cluster_size="$2"
+	local expected_num_clusters="$3"
+	local expected_num_allocated_clusters="$4"
+	local expected_fragmap="$5"
+
+	[ "$(jq '.cluster_size' <<< "$fragmap")" == "$expected_cluster_size" ]
+	[ "$(jq '.num_clusters' <<< "$fragmap")" == "$expected_num_clusters" ]
+	[ "$(jq '.num_allocated_clusters' <<< "$fragmap")" == "$expected_num_allocated_clusters" ]
+	[ "$(jq -r '.fragmap' <<< "$fragmap")" == "$expected_fragmap" ]
+}
+
+function test_fragmap_empty_lvol() {
+	# Create lvs
+	malloc_name=$(rpc_cmd bdev_malloc_create 80 $MALLOC_BS)
+	lvs_uuid=$(rpc_cmd bdev_lvol_create_lvstore "$malloc_name" lvs_test)
+
+	# Create lvol with 10 cluster
+	lvol_size=$((LVS_DEFAULT_CLUSTER_SIZE_MB * "$NUM_CLUSTERS"))
+	lvol_uuid=$(rpc_cmd bdev_lvol_create -u "$lvs_uuid" lvol_test "$lvol_size" -t)
+
+	nbd_start_disks "$DEFAULT_RPC_ADDR" "$lvol_uuid" /dev/nbd0
+
+	# Expected map: 00000000 00000000
+	fragmap=$(rpc_cmd bdev_lvol_get_fragmap $lvol_uuid)
+	verify "$fragmap" "$LVS_DEFAULT_CLUSTER_SIZE_BTYE" "$NUM_CLUSTERS" 0 "AAA="
+
+	# Stop nbd device
+	nbd_stop_disks "$DEFAULT_RPC_ADDR" /dev/nbd0
+
```

Comment:
For a cleaner exit, must delete lvstore calling `bdev_lvol_delete_lvstore` bdefore deleting the malloc bdev.
The same for all other tests below.

Reply:
Addressed. Thank you.